src/Symfony/Component/Console/Command/Command.php

```diff
@@ -159,7 +159,7 @@ protected function configure()
      *
      * @see setCode()
      */
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
```

Comment:
This is a BC break. We cannot do this before Symfony 6.

Reply:
Yep, but as I said, it was already deprecated in 4.4 : see https://github.com/symfony/symfony/pull/33775

I think it was forgotten when the switch was made.

Reply:
it was deprecated to return `null` indeed. And that's indeed forbidden in 5.1.0. But it is perfectly valid to write code in 5.1.0 which always return an int but without the return type. and that's where the BC break is.

Reply:
Fair enough. I'll wait to propose this for 6 then (if I didn't forgot :joy: ).

Reply:
we won't because there are many methods that will benefit from a similar change and are waiting for v6